src/Lean/Linter/Basic.lean

```diff
@@ -29,8 +77,9 @@ If `linterOption` is enabled, print a linter warning message at the position det
 Whether a linter option is enabled or not is determined by the following sequence:
 1. If it is set, then the value determines whether or not it is enabled.
 2. Otherwise, if `linter.all` is set, then its value determines whether or not the option is enabled.
-3. Otherwise, the default value determines whether or not it is enabled.
+3. Otherwise, if any of the linter sets containing the option is enabled, it is enabled.
```

Comment:
Disabling a linter set this option is contained in also disables the linter, right? (You have a test for this, I believe - otherwise, I think this would be useful to clarify by adding one.)
would "is enabled or explicitly disabled" be a better phrasing?

Reply:
Re-reading the test below clarifies this: disabling a linter set (even after enabling it before) means the linter set simply does not exist for the purposes of this algorithm. Perhaps it's worth noting that "enabling and disabling a linter set-again is a no-op" or something.